app/Repositories/MessageRepository.php

```diff
@@ -120,6 +120,12 @@ public function updateMessagesForContest($contest, $messages)
                 $data['type'] = $type;
                 $data['key'] = $key;
 
+                if ($data['type'] === 'leaderboard') {
+                    if (! array_key_exists('show_images', $data)) {
+                        $data['show_images'] = 0;
```

Comment:
shouldn't this be the opposite? 
```
if ($show images)
// display the images
```
which means `show_images` should be 1, not zero? 


SORRY THAT THIS IS A STREAM OF BRAIN THOUGHTS AND MIGHT NOT MAKE SENSE.